src/sections/Home/Banner/index.js

```diff
@@ -1,24 +1,25 @@
 import React, { useEffect, useState } from "react";
 import BannerDefault from "../Banner-default";
-// import BannerAlt from "../Banner-alt"; 
+// import BannerAlt from "../Banner-alt";
 
 const Banners = [
   <BannerDefault key={0} />,
-  // <BannerAlt key={1}/> 
+  // <BannerAlt key={1} /> 
 ];
 
 const Banner = () => {
-  const [Banner, setBanner] = useState(0);
+  let initialValue;
+  try {
+    initialValue = sessionStorage.getItem("banner") || 0;
+  } catch (e) {
+    initialValue = 0;
+  }
+  const [Banner, setBanner] = useState(initialValue);
   useEffect(() => {
     if (sessionStorage.getItem("banner")) {
       sessionStorage.setItem("banner", (1 + Number(Banner)) % Banners.length);
```

Comment:
Currently, we have only 2 banners and if the banner value set is `1` then here it will update to 2. Won't that throw an error?
Shouldn't there be a check on the length of the `banners` list and then update the value?